RxCocoa/iOS/UIViewController+Rx.swift

```diff
@@ -21,4 +21,54 @@
         }
     
     }
+
+    extension Reactive where Base: UIViewController {
```

Comment:
We can't add this set of extensions unfortunately.

Reply:
@kzaher out of curiosity, why so? I have a subset of these extensions in my project, are they unsafe to use?

Reply:
I don't know if that's @kzaher's reasoning, but in a previous project we had a crash report library that applied swizzling to UIViewController methods, then using Rx to check for method invocation would produce crashes. I'd say then that this usage should be project-specific instead of being part of the library, as it won't be safe for everybody to use

Reply:
We're using similar extensions in our project. Do you know in which cases it might be unsafe?